soroban-test-wasms/wasm-workspace/contract_sac_transfer/src/lib.rs

```diff
@@ -1,6 +1,6 @@
 #![no_std]
 
-use soroban_sdk::{contract, contractimpl, Env, symbol_short, IntoVal, Address,
+use soroban_sdk::{contract, contractimpl, Env, symbol_short, IntoVal, Address, Vec,
```

Comment:
We shouldn't ever update older protocol Wasms as new SDK changes their compatibility. We also don't need p21/22 ones (and it's not like they're correct here). So please only update p23 wasm.